test/tests.js

```diff
@@ -274,6 +298,13 @@ QUnit.test('with attributes', function (assert) {
 	assert.strictEqual(document.cookie, '', 'should delete the cookie');
 });
 
+QUnit.test('deletion with unofficial attributes', function (assert) {
+	assert.expect(1);
+	var attributes = { path: '/', unofficial: 'anything' };
```

Comment:
Do we actually need to pass the unofficial attributes to be able to remove the cookie or was this just following the previous examples?